packages/adapter-cloudflare-workers/index.js

```diff
@@ -11,16 +11,17 @@ import { fileURLToPath } from 'node:url';
  *   site: {
  *     bucket: string;
  *   }
+ *   compatibility_flags?: string[];
  * }} WranglerConfig
  */
 
 /** @type {import('./index.js').default} */
-export default function ({ config = 'wrangler.toml' } = {}) {
+export default function (options = { config: 'wrangler.toml' }) {
```

Comment:
Why did this change? The old code provided a default value for `config`; the new code provides a default value for` options` but _not_ for `config` specifically. As such, `options.config` could be undefined

Reply:
reverting the change

```suggestion
export default function ({ config = 'wrangler.toml' } = {}) {
```